beta/src/content/learn/passing-props-to-a-component.md

```diff
@@ -343,7 +343,7 @@ export function getImageUrl(person, size = 's') {
 
 </Sandpack>
 
-Try replacing the `<Avatar>` inside `<Card>` with some text to see how the `Card` component can wrap any nested content. It doesn't need to "know" what's being rendered inside of it. You will see this flexible pattern in many places.
+Try replacing the `Avatar` element inside the `Card` element with some text to see how the `Card` component can wrap any nested content. It doesn't need to "know" what's being rendered inside of it. You will see this flexible pattern in many places.
 
 You can think of a component with a `children` prop as having a "hole" that can be "filled in" by its parent components with arbitrary JSX. You will often use the `children` prop for visual wrappers: panels, grids, and so on.
```

Comment:
New to React and going through docs to learn, came across this mistake(I believe). 
```suggestion
You can think of a component with a `children` prop as having a "hole" that can be "filled in" by its children components with arbitrary JSX. You will often use the `children` prop for visual wrappers: panels, grids, and so on.
```